pandas/core/frame.py

```diff
@@ -227,6 +232,30 @@
 
 """
 
+def _single_replace(self, to_replace, method, inplace, limit):
```

Comment:
Could define this in one place and then import it in `frame.py` and `series.py`.

Also a short docstring would still be nice, even though it's internal.